src/proj.h

```diff
@@ -328,6 +340,17 @@ extern const PJ      *proj_shutdown;
 #endif
 
 
+/* Set or read error level */
+int  proj_errno (PJ *P);
+void proj_errno_set (PJ *P, int err);
```

Comment:
> Perhaps, but the issue is that projects.h was already a installed header (since it is include_HEADERS in 

It says in the top of `projects.h` that it is to be considered private. It was excluded from `include_HEADERS` in version 4.8, but was re-introduced with 4.9 because there where some complaints about missing functionality in `proj_api.h`. I think it is fair to remove it from the public once again and rename it `proj_internal.h`. Obviously not in the comming release, but the next as I suggested on the mailing list.